### ISSUE TITLE
Error on processing containers version 4.31

### ISSUE THREAD

**Issue by alanwjunior** [Unitech/pm2#4685]
I have a pipeline DevOps on Amazon ECS and it stopped to run succesfully yesterday. I realized that pm2 launched a new realease yesterday. I'm getting the error on ProcessContainer.js on line 68. Here is the error:
PM2 error: TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received type number (18).

The command that i use to run my application is `pm2 start --env $ENVIRONMENT --no-daemon `and my ecosystem config file has this environment config:

```
env_staging: {
          NODE_ENV: 'staging',
          NODE_PATH: '.',
          NODE_CONFIG_DIR: './config'
        },
```

The error is crashing the application and the container stop after it.

**Comment by Unitech** [comment]
Yes that's because your container is running Node.js 14, a version that was published just yersterday and there was some compatibility issues

Now it's fixed and published on PM2 4.4.0

**Comment by KevinSimple** [comment]
@Unitech  what you mean by just "published", publish means for the public to use, its sad story hahaha ,but funny  xD
